Initialized Next.js 15 in Existing Repo; Added Hero and Navbar Components

arynku opened the pull request.

arynku committed:
New Init

arynku committed:
Update NavHeader.tsx

Co-Authored-By: Brian Lam <110221313+brainlame@users.noreply.github.com>

arynku committed:
Centered Hero Section and made Logo Larger

Co-Authored-By: Brian Lam <110221313+brainlame@users.noreply.github.com>

Jolteous reviewed (commented):
Looks good to me, I'm going to let @MostOriginalIGN look at this as well, there may be some room for growth so updates in the future are more streamlined.

MostOriginalIGN reviewed (dismissed):
Looks great and organized. But no please do not push an env file if needed. It will never be needed. 

Also why did the read me change, otherwise its all good.

MostOriginalIGN commented:
Okay wait nextjs ci deleted???

MostOriginalIGN reviewed (changes requested):
**Why was everything moved into a subdirectory?**
  - Having a subdirectory unnecessarily complicates the repo structure.
  - For ease of access and better organization, we should keep the main content at the root level.

**Why was the CI check for Next.js removed?**
  - Removing the CI check might compromise our ability to catch issues early.
  - CI checks ensure the project adheres to best practices and avoids regressions. Was there a specific reason for this removal and are there any plans to replace it?

**Package Manager Consistency**
  - Just making sure because I saw some changes mixing `npm` and `yarn` can lead to dependency resolution issues.
  - Let's stick to one package manager consistently across the project to avoid potential conflicts.

**Environment Variables**
  - Committing `.env` or `.env.local` files should never be optional.
  - These files can expose sensitive or development-only configurations.
  - Instead, we should use GitHub or Vercel environment variables for managing sensitive data securely.

Sorry these things weren't evident at first for me as I was on mobile.

CryptoSOL123 committed:
Commitited INput

CryptoSOL123 committed:
Made the menotrs and robot page

arynku committed:
Moved code out of sub directory

CryptoSOL123 committed:
Finished the mentor page

Jolteous committed:
3JS Initialization + Our Robot Page GLTF Rendering

Currently both GLTF files are in the project, the roborio is being rendered right now, because the robot CAD takes way too long to render, we are making a simplified robot CAD

MostOriginalIGN committed:
Team Page

Bachthoven committed:
idk

Bachthoven committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

rqhu2025 committed:
Stuff

brainlame committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

Bachthoven committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

arynku committed:
Modified/Added a Few Pages

Co-Authored-By: Brian Lam <110221313+brainlame@users.noreply.github.com>
Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: Glue <91361677+ItsGlue@users.noreply.github.com>

arynku committed:
Modified a few pages

arynku committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

AnmingZhang committed:
Updated mentors/leadership pages

CryptoSOL123 committed:
Made the contact page

brainlame committed:
Leadership, mission statement, robot page

Co-Authored-By: Aryan Kumar <113646426+Arbitr-ary-ily@users.noreply.github.com>
Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>
Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>

MostOriginalIGN committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

MostOriginalIGN committed:
delete gltf

MostOriginalIGN committed:
fix lfs

MostOriginalIGN committed:
fix lfs again what

Bachthoven committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

MostOriginalIGN reviewed (commented):
Everything looks good so far, except for the fact it cant build. I'm gonna re add the CI just in case. Also why is this still here 
<img width="592" alt="image" src="https://github.com/user-attachments/assets/41f7f453-a087-4339-9bfc-5528c796bb82">

MostOriginalIGN committed:
Add: CI for build checks

THE-SMALL-K committed:
Update page.tsx

Changed picture of Win Htet Lin instead of Keshav's picture, edited last name of Anson and one word edit to description.

brainlame committed:
Fixed leadership images, added blog page, improvements tohome page

Co-Authored-By: Aryan Kumar <113646426+Arbitr-ary-ily@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: Bachthoven <179424948+Bachthoven@users.noreply.github.com>
Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>

THE-SMALL-K committed:
Create Outreach.tsx

Started creating the outreach page on the website under components, i'm not sure if it is the correct place to put it. I used the mission statement code as well as some of the old code from the old website and the text is from the old website. Subject to be edited both code and text.

THE-SMALL-K committed:
Create Outreach.tsx

Started creating the outreach page on the website under components, i'm not sure if it is the correct place to put it. I used the mission statement code as well as some of the old code from the old website and the text is from the old website. Subject to be edited both code and text.

VivianS27 committed:
Outreach page move under app

Copy pasted code from outreach.tsx under components to the a new page under app

Co-Authored-By: KhoiLe7 <143638032+KhoiLe7@users.noreply.github.com>

THE-SMALL-K committed:
Page.tsx fix/restored outreach.tsx

Originally put code under apps in page.tsx but then I realized I was wrong. Code should stay under components and just copy over the page.tsx code from the other pages to create a new outreach page.

brainlame committed:
Leadership page finished, uncommented robot page

THE-SMALL-K committed:
Delete page.tsx

The-Coding-Kid committed:
some changes

added some passions for mentors, addes some padding and removed a next config that caused errors

AnmingZhang committed:
Sponsors page (needs update)

added sponsors from old website (probably needs to be updated - check with business)

THE-SMALL-K committed:
Tried to fix error with running website

Deleted the imports in globals.css (was told to by Aryan Kumar) and commented out some importants and some code within HeroSection.tsx with some errors. I think running the website should work now.

THE-SMALL-K committed:
Accidentally deleted tailwind base

Tailwind base should be present i think now should work

THE-SMALL-K committed:
Website format werid/tried to fix it

Updated outreach page with some pictures. Also tried to see what the problem was with the website format.

THE-SMALL-K committed:
Update HeroSection.tsx

hero section commented out

brainlame committed:
Restoring tailwind, file cleanup

arynku committed:
Updated Home Page (Added Carousel Effect)

The-Coding-Kid committed:
more commits

changes + bugfixes

brainlame committed:
Add media page, outreach, sponsors, RIS 6 shenanigans

Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>
Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: Bachthoven <179424948+Bachthoven@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>

brainlame committed:
Some updating text and images

Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: Aryan Kumar <113646426+Arbitr-ary-ily@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>
Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>
Co-Authored-By: Bachthoven <179424948+Bachthoven@users.noreply.github.com>

THE-SMALL-K committed:
text_changes

Changed the wording of some of the text. Sounded weird.

The-Coding-Kid committed:
some changes

rqhu2025 committed:
update motion

rqhu2025 committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

brainlame committed:
Sychronized color schemes, fixing gaping errors in website

Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: Aryan Kumar <113646426+Arbitr-ary-ily@users.noreply.github.com>
Co-Authored-By: Ethan <179424948+Bachthoven@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>
Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>

AnmingZhang committed:
small changes

THE-SMALL-K committed:
more text edits

Made small word edits. There are some sections that have incomplete sentences. Will on more tomorrrow.

brainlame committed:
Synchronizing Titles, colors, more fixes

Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: Aryan Kumar <113646426+Arbitr-ary-ily@users.noreply.github.com>
Co-Authored-By: Ethan <179424948+Bachthoven@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>
Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>

The-Coding-Kid committed:
added some images to the website

brainlame committed:
Working Deployable

synchronized headers, made UI fixes, fixed bugs, efficiency

Co-Authored-By: aryan <113646426+Arbitr-ary-ily@users.noreply.github.com>
Co-Authored-By: rqhu2025 <91698182+rqhu2025@users.noreply.github.com>
Co-Authored-By: CryptoSOL123 <163804105+CryptoSOL123@users.noreply.github.com>
Co-Authored-By: Eshan41 <179530953+Eshan41@users.noreply.github.com>
Co-Authored-By: Ethan <179424948+Bachthoven@users.noreply.github.com>
Co-Authored-By: Yashveer1234 <141895202+Yashveer1234@users.noreply.github.com>

rqhu2025 committed:
wording change

rqhu2025 committed:
Merge branch 'Revamp' of https://github.com/frc-7419/7419tech into Revamp

brainlame committed:
fixing filepaths for deployment

brainlame committed:
fixing an annoying case sensitive issue by making 2 commits

brainlame committed:
Fixing case sensitive issue part 2
